src/codecoverage/bot/code_coverage_bot/codecov.py

```diff
@@ -144,7 +144,12 @@ def go_from_trigger_mozilla_central(self):
             token=secrets[secrets.COVERALLS_TOKEN]
         )
         logger.info('Report generated successfully')
-
+        json_output = json.loads(output)
+        file_list = [file['name'] for file in json_output['source_files']]
```

Comment:
No need to create a list here, you can just iterate directly in the `any` (f['name'].endswith(extension) for f in report['source_files'])